server/model/test_run_event.go

```diff
@@ -61,10 +61,9 @@ func (e TestRunEvent) ResourceID() string {
 }
 
 type PollingInfo struct {
-	Type                PollingType
-	ReasonNextIteration string
```

Comment:
We need to change this field to the `TracePollingSuccess` event, because we have reason to stop the iterations but not to run a new one.
For instance: we can stop a polling process because the trace hasn't changed for a while or due to a timeout. But we cannot say that about the iteration. Does it make sense?